spec/requests/api/v1/items/update_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe 'Items API' do
+  it 'can update an item' do
+    merchant = create(:merchant)
+    item = create(:item, merchant_id: merchant.id)
+    previous = Item.last.name
+    item_params = {name: 'Mini Fan'}
+
+    patch "/api/v1/items/#{item.id}", params: item_params
```

Comment:
Note: when comparing this with merchants spec, don't forget to change api/v1/items and not keept merchants. silly mistake created a nil class error